# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gmic/hotfix-opencv.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- gmic-4.0.3.orig/cmake/FindOpenCV.cmake	2026-07-20 13:53:30.777990860 +0200
+++ gmic-4.0.3/cmake/FindOpenCV.cmake	2026-07-20 13:53:38.030321976 +0200
@@ -1,7 +1,7 @@
 # The CMake modules shipped with OpenCV do not contain targets.
 find_package(PkgConfig)
 
-pkg_check_modules(OPENCV opencv4)
+pkg_check_modules(OPENCV opencv5)
 
 if(NOT OPENCV_FOUND)
   pkg_check_modules(OPENCV opencv)
